### PR TITLE
Aupdate on teh use of the dot wilcard character 

### PR DESCRIPTION
**Pull request by ojrb**
This PR solved the issue #4408  please review and merge !

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/windows-defender-application-guard/configure-wd-app-guard.md [hunk 1]
```diff
@@ -29,11 +29,13 @@ These settings, located at **Computer Configuration\Administrative Templates\Net
...
+|-----------|------------------|-----------|
+|Private network ranges for apps|At least Windows Server 2012, Windows 8, or Windows RT|A comma-separated list of IP address ranges that are in your corporate network. Included endpoints or endpoints that are included within a specified IP address range, are rendered using Microsoft Edge and won't be accessible from the Application Guard environment.|
+|Enterprise resource domains hosted in the cloud|At least Windows Server 2012, Windows 8, or Windows RT|A pipe-separated (\|) list of your domain cloud resources. Included endpoints are rendered using Microsoft Edge and won't be accessible from the Application Guard environment. Notes: 1) If you want to specify a complete domain, please include a full domain name **"contoso.com"** for example, in the configuration 2) You may optionally use "." as a previous wildcard character to automatically trust all subdomains (When there is more than one subdomain). Configuring **".constoso.com"** will automatically trust **"subdomain1.contoso.com"**, **"subdomain2.contoso.com"** etc. 3) To trust a subdomain, you must precede your domain with two dots, for example: **"..contoso.com"** |
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-application-guard/configure-wd-app-guard.md]:
```suggestion
|Enterprise resource domains hosted in the cloud|At least Windows Server 2012, Windows 8, or Windows RT|A pipe-separated (\|) list of your domain cloud resources. Included endpoints are rendered using Microsoft Edge and won't be accessible from the Application Guard environment. Notes: 1) If you want to specify a complete domain, include a full domain name (for example "contoso.com") in the configuration. 2) You may optionally use "." as a previous wildcard character to automatically trust all subdomains (when there is more than one subdomain). Configuring ".constoso.com" will automatically trust "subdomain1.contoso.com", "subdomain2.contoso.com", etc. 3) To trust a subdomain, precede your domain with two dots, for example "..contoso.com". |
```

**Comment by mypil** [comment]
@ojrb  - please merge the commit on the copy/edit review made by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by illfated** [comment]
Just a suggestion (you may choose to disapprove, it is not a required recommendation):

If you want some of the text in a table to start on a new line, for example to add a line break before **Notes:** and each of its 3 points/subsections, you need to use the HTML tag `<br>` within the MarkDown table (inside your edited text).
This is because MarkDown ignores ordinary NewLine (Enter) (Carriage Return/Line Feed) characters within text inside MarkDown tables.
So if you want to make the text look like this example (below), you need to use `<br>` as the functional NewLine / Line break (CR/LF) character to start each new line.

| Policy name | Supported versions | Description |
| ----------- | ------------------ | ----------- |
| Private network ranges for apps | At least Windows Server 2012, Windows 8, or Windows RT | A comma-separated list of IP address ranges that are in your corporate network. Included endpoints or endpoints that are included within a specified IP address range, are rendered using Microsoft Edge and won't be accessible from the Application Guard environment. |
| Enterprise resource domains hosted in the cloud | At least Windows Server 2012, Windows 8, or Windows RT|A pipe-separated ( \| ) list of your domain cloud resources. Included endpoints are rendered using Microsoft Edge and won't be accessible from the Application Guard environment.<br>Notes:<br>1) If you want to specify a complete domain, include a full domain name (for example "**contoso.com**") in the configuration.<br>2) You may optionally use "." as a previous wildcard character to automatically trust all subdomains (when there is more than one subdomain). Configuring "**.constoso.com**" will automatically trust "**subdomain1.contoso.com**", "**subdomain2.contoso.com**", etc.<br>3) To trust a subdomain, precede your domain with two dots, for example "**..contoso.com**". |
| Domains categorized as both work and personal | At least Windows Server 2012, Windows 8, or Windows RT | A comma-separated list of domain names used as both work or personal resources. Included endpoints are rendered using Microsoft Edge and will be accessible from the Application Guard and regular Edge environment. |

BTW, to make the pipe character ( | ) be more visible in the text, I suggest adding 1 space inside the parentheses like this: `( \| )`

**Comment by ojrb** [comment]
Ready @mypil I commited the copy-review recommendations !

**Comment by mypil** [comment]
@Dansimp - PR has been copy edited and is ready for final review. This is based on issue #4408.

Thank you.

**Pull request merged**